apps/user-portal/src/components/federated-associations/federated-associations.tsx

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Grid, Icon, List, Popup } from "semantic-ui-react";
+import { deleteFederatedAssociation, getFederatedAssociations } from "../../api/federated-associations";
+import { SettingsSectionIcons } from "../../configs";
+import {
+    AlertInterface,
+    AlertLevels,
+} from "../../models";
+import { FederatedAssociation } from "../../models/federated-associations";
+import { SettingsSection, UserAvatar } from "../shared";
+
+/**
+ * Prop types for `FederatedAssociations` component
+ */
+interface FederatedAssociationsProps {
```

Comment:
Shall we suffix the interfaces with `Interface`. Check the other places as well.